go.mod

```diff
@@ -291,6 +291,7 @@ replace (
 	github.com/Shopify/sarama => github.com/elastic/sarama v1.19.1-0.20220310193331-ebc2b0d8eef3
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
+	github.com/kardianos/service => github.com/elastic/kardianos-service v0.0.0-20230215200102-9832e01049dd
```

Comment:
Do we have a follow up issue to change the import in our codebase?

Reply:
I wasn't planning on changing the imports.  Do you think we need to? 

If we changed the imports, we could get rid of this replace directive and directly require `github.com/elastic/kardianos-service` instead.  Would we then also have to update the module name here: https://github.com/elastic/kardianos-service/blob/master/go.mod#L1?

Not saying I'm opposed to changing the imports, just wondering if we need to and, if so, what other changes might need to go hand in hand with that.